### PR TITLE
Added CODE_OF_CONDUCT.md

### LINKED ISSUE #29: Add CODE_OF_CONDUCT.md
**Is your feature request related to a problem? Please describe.**
No CODE_OF_CONDUCT.md file is present.

**Describe the solution you'd like**
@tanmayrauth @ankitshaw  As a GSSOC participant I would like to add a CODE_OF_CONDUCT.md file in this project.

**Additional context**
### Screenshots

code of conduct1
code of conduct2

### PR DESCRIPTION
**Pull request by Aman-Codes**
## Issue that this pull request solves

 Closes: #29   (issue number)

## Proposed changes
Added CODE_OF_CONDUCT.md

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by tanmayrauth** [comment]
Please add a link to this code of conduct in readme.md

**Comment by Aman-Codes** [comment]
@tanmayrauth Updated the readme and do let me know if any changes are required.

File: README.md [hunk 1]
```diff
@@ -26,3 +26,5 @@ We have the following branches
 
  * **master**
    This contains the stable code. After significant features/bugfixes are accumulated on development, we move it to master.
+   
+   All the contributers are requested to follow our [Code Of Conduct](https://github.com/Aman-Codes/travel-safe/blob/development/CODE_OF_CONDUCT.md)
```
Comment (ankitshaw) [inline README.md]: can you please add it to a separate bullet point named Code of Conduct

Comment (Aman-Codes) [reply README.md]: @ankitshaw Sir did the asked changes please review it do let me know if any changes are required.

**Pull request merged**